OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -118,6 +118,7 @@ public class UpdatePath
 			new UpdatePath("playtest-20190106", new UpdateRule[]
 			{
 				new RemoveAttackSuicides(),
+				new MakeMobilePausableConditional(),
```

Comment:
You'll need to run this rule too! There are quite a few places across all the mods that haven't been updated, mainly related to EMP in TS and transports in TS/RA/TD.

Reply:
In case you need it: `OpenRA.Utility.exe ra --update-mod MakeMobilePausableConditional --apply` (then replace `ra` with `cnc`, `d2k`, `ts`).  There will be a few unrelated changes and most of the oramap files will show as changed, but these can be ignored/discarded.

Reply:
Yeah, that was a bit silly of me. Updated across all mods now.